src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1961,6 +1961,10 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode, callable $e
                                 ->scalarNode('http_version')
                                     ->info('The default HTTP version, typically 1.1 or 2.0, leave to null for the best version.')
                                 ->end()
+                                ->booleanNode('allow_persistent_handles')
```

Comment:
this should be at the same level as max_host_connections, this cannot be configured as a client option
about the naming, what about this?

```suggestion
                                ->booleanNode('use_persistent_connections')
```